tests/utils/decorators.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+skip_if_maintenance_mode = pytest.mark.skipif(
+    pytest.config.getoption('--maintenance-mode'),
+    reason='because the target server is in maintenance mode'
```

Comment:
Can we drop "because"?  It looks like this in the output, would be a little shorter:

```
====================================================== short test summary info ======================================================
SKIP [1] tests/functional/test_notfound.py:20: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_article_revision.py:28: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_article_revision.py:7: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_search.py:117: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_article.py:46: because the target server is not in maintenance mode
SKIP [65] tests/functional/test_maintenance_mode_redirects.py:11: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_home.py:76: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_dashboard.py:42: because the target server is not in maintenance mode
SKIP [1] tests/functional/test_profiles.py:7: because the target server is not in maintenance mode
```